### DISCUSSION TITLE
Option to Prevent Infinite Lava with Dripstone

### DISCUSSION THREAD

**Discussion by StrangeOne101** [PurpurMC/Purpur#662] [__Completed__] [answered]
In 1.17, dripstone can be used to drip lava into cauldrons, creating infinite lava.

It would good to be able to disable this behaviour. I think it could be done in 2 ways:

1. Disable the dripstone filling the cauldron with lava completely
2. Turn the lava block above the dripstone to air when the cauldron is filled with lava (to the max level). This would prevent infinite lava but still allow for the drip feature to exist

Thanks :)

**Comment by encode42** [comment, marked as answer]
This is now possible by setting [`fill-chances.dripstone-lava`](https://purpurmc.org/docs/Configuration/#dripstone-lava) to `0`.
